slpfaucet.ts

```diff
@@ -217,7 +234,6 @@ export class SlpFaucetHandler {
     }
 
     public async nftChildTokenSend(groupId: string, inputUtxos: slpjs.SlpAddressUtxoResult[], tokenReceiverAddress: string, changeReceiverAddress: string): Promise<string> {
```

Comment:
on line 240, it should be `const documentHash: Buffer|null = Buffer.from(process.env.DOCUMENTHASH!, "hex");`.   "hex" is missing.

Reply:
fixed in commit# 54a464f